includes/class-sensei-learner.php

```diff
@@ -99,6 +99,191 @@ public function delete_all_user_activity( $user_id = 0 ) {
 		return $dataset_changes;
 	}
 
+	/**
+	 * Query the courses a user is enrolled in.
+	 *
+	 * @param int   $user_id         User ID.
+	 * @param array $base_query_args Base query arguments.
+	 *
+	 * @return WP_Query
+	 */
+	public function get_enrolled_courses_query( $user_id, $base_query_args = [] ) {
+		$this->before_enrolled_courses_query( $user_id );
+
+		$query_args = $this->get_enrolled_courses_query_args( $user_id, $base_query_args );
+
+		return new WP_Query( $query_args );
+	}
+
+	/**
+	 * Query the courses a user is enrolled in and hasn't completed.
+	 *
+	 * @param int   $user_id         User ID.
+	 * @param array $base_query_args Base query arguments.
+	 *
+	 * @return WP_Query
+	 */
+	public function get_enrolled_active_courses_query( $user_id, $base_query_args = [] ) {
```

Comment:
It felt weird as a `bool`  and generally try to avoid string `enum` arguments in public methods. I merged most functionality into one private method and then called the argument on that method. Would that work okay? d406fd8a5782a9e52788b8906c315bfba9184239